src/Sulu/Bundle/SnippetBundle/Tests/Functional/Controller/SnippetControllerTest.php

```diff
@@ -267,7 +272,7 @@ public function testDeleteReferenced()
     private function loadFixtures()
     {
         // HOTELS
-        $req = ContentMapperRequest::create()
+        $request = ContentMapperRequest::create()
```

Comment:
We can now start pro-actively replacing things like this with the document manager:

````
$doc = $documentManager->create('snippet');
$doc->setStructureType('hotel');
$doc->getTitle('Le frane..');
$documentManager->persist($doc, 'de');
$documentManager->flush();
````

But might be better to make that policy after the merge.